docs/formats.rst

```diff
@@ -7,6 +7,22 @@ Formats
 Tablib supports a wide variety of different tabular formats, both for input and
 output. Moreover, you can :ref:`register your own formats <newformats>`.
 
+cli
+====
+
+The ``cli`` format is currently export-only. The exports produce a representation
+table suited to a terminal.
+
+When exporting to a cli you can pass the table format  with the ``tablefmt``
+parameter, the supported formats are the same supported by Pretty-print tabular
+data python-tabulate. For example::
```

Comment:
"the supported formats are the same supported by Pretty-print tabular
data python-tabulate"

I don't quite understand this bit, please could you explain or reword?